ramls/examples/instancesBatch_post.json

```diff
@@ -24,7 +24,7 @@
     },
     {
       "title": "Open Bibliography for Science, Technology and Medicine",
-      "source": "MARC",
+      "source": "MARC_BIB",
```

Comment:
Is there an issue for changing all of the existing instance records that have a source of `MARC` to `MARC_BIB` during an upgrade process?

Reply:
I discussed with PO and as result - we don't need to separate source by record type. I reverted changes related to source. Only dataType was changed from MARC to MARC_BIB.

Thanks for comment!